akka-http-core/src/main/java/akka/http/javadsl/model/headers/LanguageRange.java

```diff
@@ -5,11 +5,11 @@
 package akka.http.javadsl.model.headers;
 
 public interface LanguageRange {
-    public abstract String primaryTag();
-    public abstract float qValue();
-    public abstract boolean matches(Language language);
-    public abstract Iterable<String> getSubTags();
-    public abstract LanguageRange withQValue(float qValue);
+    String primaryTag();
+    float qValue();
+    boolean matches(Language language);
+    Iterable<String> getSubTags();
+    LanguageRange withQValue(float qValue);
```

Comment:
they're public (welcome to java (method declarations in in interface)).

I would like to keep the modifiers, as we have many scala contributors, and after some years of Scala one simply forgets what "default" means in some places in java. 
So thanks for the PR but I'd like to keep them (i.e. not accept the PR), thanks!